crates/bevy_dev_tools/src/ui_debug_overlay/mod.rs

```diff
@@ -27,8 +27,6 @@ mod inset;
 
 /// The [`Camera::order`] index used by the layout debug camera.
 pub const LAYOUT_DEBUG_CAMERA_ORDER: isize = 255;
-/// The [`RenderLayers`] used by the debug gizmos and the debug camera.
-pub const LAYOUT_DEBUG_LAYERS: RenderLayers = RenderLayers::none().with(16);
```

Comment:
I couldn't figure out how to make a const version of `with`, but I agree that we probably need some way to construct a const layer with none.

Reply:
I just did it with RenderLayers::layer I think, one of the constructors is definitely const

Reply:
I added a `none_with` constructor that is const.

Reply:
unless i'm missing something it's the same code as `layer` just above it.

Reply:
by default `layer_info` will include layer 0 as true, so i just pulled it out to shift into 0 instead of 1.

Reply:
added a test to demonstrate it works

Reply:
```rust
    assert_eq!(RenderLayers::layer(5), RenderLayers::none_with(5));
```
```
 left: RenderLayers([5])
 right: RenderLayers([])
```

the original doesn't include layer 0, and the new one doesn't work - `0 << bit_index` is still zero.

Reply:
yup my bad! bit shifting confuses me. this needs to be fixed so that `layer` always includes 0.

Reply:
> this needs to be fixed so that `layer` always includes 0

i don't think it should always include layer 0 (bits 0b00...001).

we don't want the debug layers defined in this overlay tool to include layer 0 at all, we want exactly RenderLayers([16]) which is what `RenderLayers::layer(16)` will give you. that should be equivalent to the old code `RenderLayers::none().with(16)`.

layer 0 (bits 0b1) is the *default* but that doesn't mean any of the other functions should always include that layer.

Reply:
> we don't want the debug layers defined in this overlay tool to include layer 0 at all, we want exactly RenderLayers([16]) which is what RenderLayers::layer(16) will give you. that should be equivalent to the old code RenderLayers::none().with(16).

yeah, i was getting totally turned around here by the fact that they constructed this like `none().with(16)`, even on `main` they could have just done `layer(16)`. what we're actually missing is `default().with(16)`, which I think is okay to leave out for now? sorry for the mix up

Reply:
yeah i don't think we need a const fn for that.